tools/bazel.rc

```diff
@@ -1 +1,6 @@
-build --copt=--std=c++14 -c opt
+build --copt=--std=c++14
+
+build --copt=-march=corei7 --copt=-mtune=core-avx2
+
+build --workspace_status_command=tools/workspace_status.sh
```

Comment:
I didn't see this file, did I miss it?

Reply:
Yes, I forgot to git add this. I'll clean up the branch and force push it to update the pull request.